Configs/Bitwarden-Release.xcconfig

```diff
@@ -5,6 +5,8 @@ CODE_SIGN_IDENTITY = Apple Development
 #include? "./Local.xcconfig"
 
 ASSETCATALOG_COMPILER_APPICON_NAME = $(APPICON_NAME)
+BITWARDEN_FLAGS = DEBUG_MENU
```

Comment:
🤔 Is the intention that this would be overridden with some other flags locally on some CI builds? How would we, ideally, know whether or not a build on CI should or should not have the debug menu at build time?